src/libs/context.js

```diff
@@ -438,7 +438,8 @@ function blurHandler () {
  * Bound listeners.
  */
 function listeners () {
-  const { input, ...listeners } = this.$listeners
+  const { "update:modelValue": input, ...listeners } = this.$attrs
```

Comment:
`$listeners` is no longer a thing, so this sort of thing will need to be done throughout. And instead of just `$this.$attrs` I think we need to pull out only those attributes that start with `on`.